satellite.py

```diff
@@ -34,86 +34,140 @@
 
 """
 import argparse
+import math
 import dimod
 import itertools
 import json
 import neal
 import sys
 from dwave.system import LeapHybridSampler
+import matplotlib.pyplot as plt
 
+def read_in_args(args):
+    """ Read in user specified parameters."""
+
+    parser = argparse.ArgumentParser(description='Satellites example')
+    parser.add_argument('file', metavar='file', type=str, help='Input file')
+    parser.add_argument('solver', metavar='solver', type=str, help='Solver')
+    parser.add_argument('--viz', type=bool, help='Visualization', default=True)
```

Comment:
I don't think this will work. I believe you'll have to use `action` with `store_true`/`store_false`. See [here](https://stackoverflow.com/questions/15008758/parsing-boolean-values-with-argparse) for an example. Or switch to `click` for a nicer API. :wink:

Reply:
Seems like this was removed by accident? Just ran the demo and got `AttributeError: 'Namespace' object has no attribute 'viz'`